pkg/config/types.go

```diff
@@ -179,4 +179,16 @@ type Config struct {
 	DynamicFilePath string
 	//use UserId for mapping, IdentityArn is not used any more when DynamicFileUserIDStrict=true
 	DynamicFileUserIDStrict bool
+	//reserved prefix config for username
```

Comment:
Nit: Go comments typically begin with a space, between the comment and the text. Also linters like https://golangci-lint.run/ will flag this.

Also, is the string key of the map the backend name?

```suggestion
	// ReservedPrefixConfig defines reserved username prefixes for each backend
```